main_routers/live2d_router.py

```
@@ -966,3 +966,110 @@ def open_model_directory(model_name: str):
+        is_user_model = False
+        
+        # 检查是否在用户文档目录下
+        try:
+            from utils.config_manager import get_config_manager
+            config_mgr = get_config_manager()
+            user_live2d_dir = str(config_mgr.live2d_dir).replace('\\', '/')
+            model_dir_normalized = model_dir.replace('\\', '/')
+            if model_dir_normalized.startswith(user_live2d_dir):
+                is_user_model = True
+        except Exception:
+            pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)